CHANGELOG.md

```diff
@@ -16,18 +16,23 @@ Here is a template for new release sections
 ### Removed
```

Comment:
We need to copy-paste the ## [Unreleased] to keep writing in the changelog for the future release (the one after this one)

Reply:
Fixed in d52b06d76fce252a04f84fd6ca7c4007889e388a